src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -59,6 +70,65 @@ internal void InitializeLicense(string licenseText)
             license = foundLicense;
         }
 
+        static bool FirstTimeUser()
+        {
+            // Check for the presence of HKCU:Software/NServiceBus
+            using (var regNsbRoot = Registry.CurrentUser.OpenSubKey(@"Software\NServiceBus"))
+            {
+                if (regNsbRoot == null)
+                {
+                    //Check for the presence of HKCU:SOFTWARE\ParticularSoftware
+                    using (var regParticularRoot = Registry.CurrentUser.OpenSubKey(@"Software\ParticularSoftware"))
+                    {
+                        if (regParticularRoot == null)
+                        {
+                            return true;
+                        }
+                    }
+                }
+            }
+            return false;
+        }
+
+
+        private void TrackFirstTimeUsageEvent()
+        {
+            // Get the current version of NServiceBus that's being used
+            var assembly = Assembly.GetExecutingAssembly();
+            var version = assembly.GetName().Version.ToString();
+
+            if (assembly.Location != null)
+            {
+                version = FileVersionInfo.GetVersionInfo(assembly.Location).FileVersion;
+            }
+
+            try
+            {
+                var azureFuncUrl =
```

Comment:
@SeanFeldman @kbaley - I discussed this with @gbiellem and we think that it is better as part of the Particular.net backend api, in which case the domain will already be there and is since its already hosted, it is the same as adding the Azure function. So, I am going to move this to the backend.